src/guide/migration/filters.md

```diff
@@ -75,17 +75,17 @@ Instead of using filters, we recommend replacing them with computed properties o
 
 ### Global Filters
 
-If you are using filters that were globally registered and then used throughout our app, it's likely not convenient to replace them with computed props or methods in each individual component.
+If you are using filters that were globally registered and then used throughout your app, it's likely not convenient to replace them with computed properties or methods in each individual component.
```

Comment:
The sentence starts off with 'you', so I've updated the 'our' to 'your'.

'Computed properties' needs to be written out in full.

Reply:
Good improvement!